examples/process/www/components/Container.react.js

```diff
@@ -0,0 +1,103 @@
+import React, { PropTypes, PureComponent } from 'react';
```

Comment:
ProcessManager should return list of tasks. Task "start"  and "sendEvent" should be done inside the ProcessManager, e.g. "update" takes place after successful event sending inside ProcessManager.